transform/transformers/common_software/pck_transformer.py

```diff
@@ -28,6 +28,104 @@ class PCKTransformer:
     # The list of answers that DON'T need rounding is much shorter.
     qsi_non_currency_questions = ["11", "12", "15", "146", '146a', '146b', '146c', '146d', '146e', '146f', '146g', '146h']
 
+    # Mapping used to calculate totals and which qcode should hold the total value.
```

Comment:
Minor point , this section of constants in the PCK transformer is getting large. Most of the variable names , like the one you are adding here, have the survey as a prefix , but some of the older ones do not(not this pr).I can only see this section getting larger over time , so I think we need to find a way to make it more obvious to subsequent developers what the values are . Could be we even consider loading these from files , or we add more documentation in the code.